Lullabot/lb-ebook-poc
# 56 Scrolling Content

Resolves #56 

This PR ensures only the main text area scrolls and not the sidebar or image containers. Additionally, updates dependencies and adds links to each available ebook when running the app on a local, since the bare server url now automatically redirects away from where you need to be.

To test:

- visit https://pr57-hwex9jpj3mqbvksmb6wq5oyglugldama.tugboatqa.com/ebooks/large-team-challenges/cover/
- page through until reaching a chapter with an image
- ensure that only the main center text area scrolls and not the sidebar or image container
